types/address.go

```diff
@@ -238,81 +206,50 @@ func (aa *AccAddress) Unmarshal(data []byte) error {
 	return nil
 }
 
-// MarshalJSON marshals to JSON using Bech32.
+// MarshalJSON marshals to JSON using hex.
 func (aa AccAddress) MarshalJSON() ([]byte, error) {
 	return json.Marshal(aa.String())
 }
 
-// MarshalYAML marshals to YAML using Bech32.
+// MarshalYAML marshals to YAML using hex.
 func (aa AccAddress) MarshalYAML() (interface{}, error) {
 	return aa.String(), nil
 }
 
-// UnmarshalJSON unmarshals from JSON assuming Bech32 encoding.
+// UnmarshalJSON unmarshals from JSON assuming hex encoding.
 func (aa *AccAddress) UnmarshalJSON(data []byte) error {
 	var s string
-	err := json.Unmarshal(data, &s)
-	if err != nil {
-		return err
-	}
-	if s == "" {
-		*aa = AccAddress{}
-		return nil
-	}
```

Comment:
Is this special case handled by `common.HexToAddress`? (I looked into it and it seems so)

Reply:
Could not find this special case in `common.HexToAddress`.
In that case, I'll implement it.

Reply:
Implemented in latest commit

Reply:
Resolved